.github/workflows/flake8.yml

```diff
@@ -30,6 +30,7 @@ jobs:
       run: |
         sudo apt-get update -qy
         sudo apt-get install -y python3-dev python3-pip meson ninja-build
+        python3 -m pip install ContactMechanics
```

Comment:
I think I have to paddle back on my earlier comment here. If we put this in testing, we may run into weird problems with circular dependencies. I would prefer not to have downstream project in here.